auth/authjwt/authjwt.go

```diff
@@ -12,47 +12,49 @@ import (
 	"gopkg.in/gomail.v2"
 
 	"github.com/auth_server/db"
-	_ "github.com/lib/pq"
+	_ "github.com/lib/pq" //Postgres Driver
 
 	jwtmiddleware "github.com/auth0/go-jwt-middleware"
 	jwt "github.com/dgrijalva/jwt-go"
 )
 
+//User information
 type User struct {
-	Id               int    `json:"id"`
-	Name             string `json:"name"`
-	Mail             string `json:"mail"`
-	password         string
-	Info_date        string         `json:"info_date"`
-	Photo            string         `json:"photo,omitempty"`
-	Github_user      string         `json:"github_user,omitempty"`
-	photo_null       sql.NullString `json:"photo"`
-	github_user_null sql.NullString `json:"github_user"`
+	ID             int    `json:"id"`
+	Name           string `json:"name"`
+	Mail           string `json:"mail"`
+	Password       string
```

Comment:
this is variable is local. Not showing in final json.

Reply:
Fixed on #14853a5